src/main/java/hillel/spring/doctor/DoctorRestAPIController.java

```diff
@@ -0,0 +1,99 @@
+package hillel.spring.doctor;
+
+
+import hillel.spring.doctor.dto.DoctorDtoConverterImpl;
+import hillel.spring.doctor.dto.DoctorInputDto;
+import hillel.spring.doctor.dto.DoctorOutputDto;
+import lombok.val;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import static org.springframework.http.HttpStatus.NO_CONTENT;
+
+
+@RestController
+public class DoctorRestAPIController {
+    private final DoctorRestAPIService doctorRestAPIService;
```

Comment:
service should have nothing in common with REST API, so please rename service class and field